src/qml/main.qml

```diff
@@ -118,6 +121,8 @@ Kirigami.ApplicationWindow {
 
 		// push roster page (trying normal start up)
 		pageStack.push(rosterPage)
+		if (!Kirigami.Settings.isMobile)
```

Comment:
The problem with this is that `isMobile` is also true on tablets. It's ok for now, I think, but it would be nicer if the page were only displayed when the screen has enough space.